jina/peapods/runtimes/zmq/zed.py

```diff
@@ -129,9 +130,9 @@ def _handle(self, msg: 'Message') -> 'ZEDRuntime':
         """Register the current message to this pea, so that all message-related properties are up-to-date, including
         :attr:`request`, :attr:`prev_requests`, :attr:`message`, :attr:`prev_messages`. And then call the executor to handle
         this message if its envelope's  status is not ERROR, else skip handling of message.
+
```

Comment:
Maybe we can add also what's returned here?

Reply:
Done